tests/unit-tests/test_construct_graph.py

```diff
@@ -229,7 +231,84 @@ def test_map_node_ids():
     check_map_node_ids_src_not_exist(str_src_ids, str_dst_ids, id_map)
     check_map_node_ids_dst_not_exist(str_src_ids, str_dst_ids, id_map)
 
+def test_convert2ext_mem():
+    # This is to verify the correctness of ExtMemArrayConverter
+    converters = [ExtMemArrayConverter(None, 0),
+                  ExtMemArrayConverter("/tmp", 2)]
+    for converter in converters:
+        arr = np.array([str(i) for i in range(10)])
+        em_arr = converter(arr, "test1")
+        np.testing.assert_array_equal(arr, em_arr)
+
+        arr = np.random.uniform(size=(1000, 10))
+        em_arr = converter(arr, "test2")
+        np.testing.assert_array_equal(arr, em_arr)
```

Comment:
You can also check whether the file exists before converter.cleanup() and whether the file is deleted after converter.cleanup().

Reply:
this is difficult. i don't know their file names.